build.sh

```diff
@@ -35,6 +35,21 @@ configure_yum_repos() {
     # can depend on those latest tools being available in all container
     # builds.
     echo -e "[f${version_id}-coreos-continuous]\nenabled=1\nmetadata_expire=1m\nbaseurl=https://kojipkgs.fedoraproject.org/repos-dist/f${version_id}-coreos-continuous/latest/\$basearch/\ngpgcheck=0\nskip_if_unavailable=False\n" > /etc/yum.repos.d/coreos.repo
+
+    # The Fedora 35 repo got archived, we need to change it to the new location
+
+    if [ $arch = "x86_64" ] || [ $arch = "aarch64" ]; then
+        sed -i "/metalink*/c\baseurl=https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/\$releasever/Everything/\$basearch/os/" /etc/yum.repos.d/fedora.repo
```

Comment:
just an FYI: this is replacing all instances of metalink in the file, which just happens to work because we don't have any repos other than the first one in each of these files enabled.